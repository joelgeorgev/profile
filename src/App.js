import React from 'react'
import 'tachyons/css/tachyons.min.css'

import { Name, Social } from './components'

export const App = () => {
  return (
    <main role='main' className='bg-image bg-center cover'>
      <Name />
      <Social />
    </main>
  )
}