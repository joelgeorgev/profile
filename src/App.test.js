import React from 'react'
import { render, unmountComponentAtNode } from 'react-dom'

import { App } from './App'

describe('App', () => {
  beforeEach(() => {
    window.IntersectionObserver = jest.fn(() => ({
      observe: () => {},
      disconnect: () => {}
    }))
  })

  afterEach(() => {
    jest.resetAllMocks()
  })

  test('renders', () => {
    const container = document.createElement('div')
    render(<App />, container)
    unmountComponentAtNode(container)
    container.remove()
  })
})
