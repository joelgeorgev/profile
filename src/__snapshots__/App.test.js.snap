// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<main
  className="bg-image bg-center cover"
  role="main"
>
  <section
    className="flex flex-column justify-center w-100 vh-100 pv4"
  >
    <div
      className="name  flex flex-auto flex-column 
      justify-center items-center white"
    >
      <div
        className="f2"
      >
        Joel George V
      </div>
    </div>
    <div
      className="down  flex justify-center"
    >
      <img
        alt="Down"
        src="down.svg"
      />
    </div>
  </section>
  <section
    className="flex flex-column justify-center w-100 vh-100 pv4"
  >
    <div
      className="appear  flex flex-column justify-center 
      items-center white ma4"
    >
      <div
        className="f3"
      >
        Say Hi!
      </div>
    </div>
    <div
      className="flex justify-center"
    >
      <a
        className="mh1 github "
        href="https://github.com/joelgeorgev"
      >
        <img
          alt="GitHub"
          src="github.svg"
        />
      </a>
      <a
        className="mh1 instagram "
        href="https://www.instagram.com/joelgeorgev"
      >
        <img
          alt="Instagram"
          src="instagram.svg"
        />
      </a>
      <a
        className="mh1 twitter "
        href="https://twitter.com/joelgeorgev"
      >
        <img
          alt="Twitter"
          src="twitter.svg"
        />
      </a>
    </div>
  </section>
</main>
`;
